### ISSUE TITLE
Clarification on Training Loss Computation with One-Step Diffusion and LR Latents Input

### ISSUE THREAD

**Issue by yyang181** [zhengchen1999/DOVE#10]
Hi, 

thank you for your excellent work on this project — it's quite impressive and inspiring!

After reviewing the paper and codebase, I have a few questions regarding the training details, specifically on how the loss is computed in your one-step diffusion setup:

From what I understand:

- You adopt a **one-step diffusion** strategy;
- The input to the diffusion model is **low-resolution (LR) video**, and the DiT backbone directly consumes the **LR latents** without adding noise;
- You seem to bypass the standard multi-step denoising schedule.

Given that, I’d like to confirm:

👉 During training, is the loss computed by directly comparing the DiT output with the ground-truth latent (from VAE)?
Or is it done via something like the following:

```python
model_pred = scheduler.get_velocity(model_output, noisy_video_latents, timesteps)
loss = F.mse_loss(model_pred, target_latents)
```

In other words — is the loss applied to the raw model output, or to the processed output after applying the scheduler's velocity / denoising function?

This detail would be very helpful for reproducing the training and aligning the loss formulation with the inference pipeline.

Thanks again for releasing your code and for any clarifications you can provide!
